stable/registry-creds-handler/templates/deployment.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "registry-creds-handler.fullname" . }}
+  labels:
+    {{- include "registry-creds-handler.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "registry-creds-handler.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "registry-creds-handler.selectorLabels" . | nindent 8 }}
+    spec:
+    {{- with .Values.image.pullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      containers:
```

Comment:
since this app allow reading envvars, allow it to read env from secret. use https://kubernetes.io/docs/concepts/configuration/secret/#use-case-as-container-environment-variables as a reference to the use case